cmd/scollector/collectors/dotnet_windows.go

```diff
@@ -0,0 +1,99 @@
+package collectors
+
+import (
+	"regexp"
+	"strings"
+	"time"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+var regexesDotNet = []*regexp.Regexp{}
+
+func init() {
+	AddProcessDotNetConfig = func(line string) error {
+		reg, err := regexp.Compile(line)
+		if err != nil {
+			return err
+		}
+		regexesDotNet = append(regexesDotNet, reg)
+		return nil
+	}
+
+	WatchProcessesDotNet = func() {
+		if len(regexesDotNet) == 0 {
+			// if no processesDotNet configured in config file, use this set instead.
+			regexesDotNet = append(regexesDotNet, regexp.MustCompile("^w3wp"))
+		}
+		c := &IntervalCollector{
+			F:        c_dotnet_loading,
+			Interval: time.Second * 60,
+		}
+		c.init = wmiInit(c, func() interface{} { return &[]Win32_PerfRawData_NETFramework_NETCLRLoading{} }, ``, &dotnetLoadingQuery)
```

Comment:
Backticks are generally only used when you need "raw string" behavior, just use double quotes. Also I would make the anon func span multiple lines